tools/apoc/apoc.xml

```diff
@@ -0,0 +1,266 @@
+                                                -lq queries_list
+                                            #else
+                                                #if $query_source.pq
+                                                    -pq $query_source.pq
+                                                #end if 
+                                            $query_source.pdbfile2
+                                            #end if
+                                            > $output_apoc
+        ]]>
+    </command>
+    <inputs>
+        <conditional name="templates_source">
+            <param name="template_source_select" type="select" label="Chose template source: pdb file or file with list of templates">
+                <option value="pdbfile">pdb file</option>
+                <option value="list">file with list of templates</option>
+            </param>
+            <when value="pdbfile">
+                <param name="pdbfile1" type="data" format="pdb" label="First (template) structure for comparison" help="(pdbfile1)" />
+                <param name="pt" type="text" label="Names of pockets in the first (template) structure for comparison" optional="true" help="(-pt)" />
+            </when>
+            <when value="list">
+                <param name="lt" type="data" format="data" label="List of templates to compare in a file" help="(-lt)" />
+                <repeat name="template" title="pdb file from list of templates">
+                    <param name="input" type="data" format="pdb" label="pdbfile" help="Load all pdb files in the same order as in the list of templates" />
+                </repeat>
+            </when>
+        </conditional>
+        <conditional name="query_source">
+            <param name="query_source_select" type="select" label="Chose query source: pdb file or file with list of queries">
+            <option value="pdbfile">pdb file </option>
+            <option value="list">file with list of queries</option>
+            </param>
+            <when value="pdbfile">
+                <param name="pdbfile2" type="data" format="pdb" label="Second (query) structure for comparison" help="(pdbfile2)" />
+                <param name="pq" type="text" label="Names of pockets in the second (query) structure for comparison" optional="true" help="(-pq)" />
+            </when>
+            <when value="list">
+                <param name="lq" type="data" format="data" label="List of queries (targets) to compare in a file." help="(-lq)" />
+                <repeat name="query" title="pdb file from list of queries">
+                    <param name="input" type="data" format="pdb" label="pdbfile" help="Load all pdb files in the same order as in the list of queries" />
+                </repeat>
+            </when>
+        </conditional>
+        <param name="fa" type="select" label="Global structure alignment" help="(-fa)">
```

Comment:
You can use the `argument` attribute instead of specifying the option of the underlying tool inside `help`. When the `name` attribute is derived from it by just removing the initial dashes, you can also omit it. So that could be rewritten as:
```xml
<param argument="-fa" type="select" label="Global structure alignment">
```

The same applies for all `<param>`s below.